ReactAndroid/src/main/jni/first-party/yogajni/jni/ScopedGlobalRef.h

```diff
@@ -113,17 +115,21 @@ class ScopedGlobalRef {
   /**
    * Returns the underlying JNI global reference.
    */
-  T get() const { return mGlobalRef; }
```

Comment:
Dirsynced projects like yoga and JSI are internally exempted from the formatter being run. E.g. Yoga has its own config at `xplat/yoga/.clang-format` which is used for Yoga itself, then formatting does not run on the RN mirror.

Reply:
Here's the list of ignored directories:

```
      "xplat/js/react-native-github/ReactAndroid/src/main/java/com/facebook/yoga/**", # Dirsync
      "xplat/js/react-native-github/ReactAndroid/src/main/jni/first-party/yogajni/**", # Dirsync
      "xplat/js/react-native-github/ReactCommon/jsi/jsi/**", # Dirsync
      "xplat/js/react-native-github/ReactCommon/yoga/**", # Dirsync
```